source/includes/_security.md

```diff
@@ -8,18 +8,405 @@ Please refer to our [RBAC manual](https://docs.killbill.io/latest/user_managemen
 
 ### List user permissions
 
+```shell
+curl -v \
+     -u admin:password \
+     -H "X-Killbill-ApiKey: bob" \
+     -H "X-Killbill-ApiSecret: lazar" \
+     -H "Accept: application/json" \
+    "http://localhost:8080/1.0/kb/security/permissions"
+```
+
+```java
+import org.killbill.billing.client.api.gen.SecurityApi;
+import org.killbill.billing.client.model.Strings;
+
+SecurityApi securityApi = new SecurityApi(killBillHttpClient);
+
+Strings currentUserPermissions = securityApi.getCurrentUserPermissions(requestOptions);
```

Comment:
Typo here: should be `String currentUserPermissions`

Reply:
> Typo here: should be `String currentUserPermissions`

Hi @reshmabidikar this looks like the correct one

![image](https://github.com/killbill/slate/assets/75751774/f0214849-8043-4dfd-a725-771e9ec8d642)